File: app/src/main/java/com/babestudios/companyinfouk/data/model/persons/Person.java
package com.babestudios.companyinfouk.data.model.persons;


import com.google.gson.annotations.SerializedName;

import java.util.ArrayList;
import java.util.List;

public class Person {
	@SerializedName("notified_on")
	public String notifiedOn;
	@SerializedName("kind")
	public String kind;
	@SerializedName("country_of_residence")
	public String countryOfResidence;
	@SerializedName("etag")
	public String etag;
	@SerializedName("date_of_birth")
	public DateOfBirth dateOfBirth;
	@SerializedName("address")
	public Address address;
	@SerializedName("links")
	public Links_ links;
	@SerializedName("name_elements")
	public NameElements nameElements;
	@SerializedName("natures_of_control")
	public List<String> naturesOfControl = new ArrayList<>();
	public String nationality;
	public String name;
	public Identification identification;
}

File: app/src/main/java/com/babestudios/companyinfouk/data/model/persons/Person.kt
package com.babestudios.companyinfouk.data.model.persons


import com.google.gson.annotations.SerializedName

import java.util.ArrayList

class Person {
	@SerializedName("notified_on")
	var notifiedOn: String? = null
	@SerializedName("kind")
	var kind: String? = null
	@SerializedName("country_of_residence")
	var countryOfResidence: String? = null
	@SerializedName("etag")
	var etag: String? = null
	@SerializedName("date_of_birth")
	var dateOfBirth: DateOfBirth? = null
	@SerializedName("address")
	var address: Address? = null
	@SerializedName("links")
	var links: Links_? = null
	@SerializedName("name_elements")
	var nameElements: NameElements? = null
	@SerializedName("natures_of_control")
	var naturesOfControl: List<String> = ArrayList()
	var nationality: String? = null
	var name: String? = null
	var identification: Identification? = null
}
